website/docs/suggestion/main-cc.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 2
+title: Main CC
+---
+
+Main suggestion command, provides CRUD functionality for suggestions and admin commands to manage suggestions.
```

Comment:
maybe write it in whole words?
I had to google what `CRUD` even means

Reply:
Will rewrite.

Reply:
epico